src/datasets/packaged_modules/audiofolder/audiofolder.py

```diff
@@ -0,0 +1,68 @@
+from dataclasses import dataclass
+from typing import List
+
+import datasets
+
+from ..folder_builder import folder_builder
+
+
+logger = datasets.utils.logging.get_logger(__name__)
+
+
+@dataclass
+class AudioFolderConfig(folder_builder.FolderBuilderConfig):
+    """Builder Config for AudioFolder."""
+
+    drop_labels: bool = True  # usually we don't need labels as classification is not the main audio task
```

Comment:
well I don't have a strong opinion here anymore :D 
if we set `drop_labels=None` by default as you suggested, it might be confusing in cases when users provide only audio files, without metadata (or with broken metadata?). this is probably quite unlikely, so I'm ok with your suggestion.

Reply:
@mariosasko what do you think about that?

Reply:
It's good to be consistent, so I agree with @lhoestq.

Reply:
i'm ok with that but it makes explaining things in documentation a bit more complicated...

Reply:
i've updated the docs, tried to make it simpler. still not sure that this logic with default None value of "drop_labels" is clear but I guess we'd better see what users say.

Reply:
@lhoestq @mariosasko what do you think about it now? 🤗 
also, don't you know what's happening with the CI? why it takes forever and finally some jobs are cancelled?

Reply:
Looks all good to me ! :D 

Not sure what's happening with the CI though. I just re-launched one job to see if it was caused by a bug in github actions or the windows runners